### PR TITLE
Dispose() method implemented

### PR DESCRIPTION
**Pull request by garciaolais**
Hi

I noticed that the method is empty, so I add this little fix

### CODE REVIEW

File: src/devices/Si7021/Si7021.cs [hunk 1]
```diff
@@ -59,5 +55,15 @@ public double ReadHumidity()
             
             return humidity;
         }
+
+        public void Dispose()
+        {
+            if (_i2cDevice != null)
```
Comment (johnastaf) [inline src/devices/Si7021/Si7021.cs]: _i2cDevice?.Dispose() ? :-)

Comment (garciaolais) [reply src/devices/Si7021/Si7021.cs]: I like your propose

Comment (garciaolais) [reply src/devices/Si7021/Si7021.cs]: > _i2cDevice?.Dispose() ? :-)

@johnastaf  done 👍

**Comment by joperezr** [comment]
Thanks @garciaolais !

**Pull request merged**